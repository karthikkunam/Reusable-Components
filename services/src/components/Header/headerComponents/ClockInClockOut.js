import React from 'react';
import { clockInAndOut } from '../../../constants';

function ClockInClockOut() {
  return (
    <form className="form-inline logged-out">
      <button type='submit' className="btn btn-sm btn-outline-primary clockin" disabled>{clockInAndOut}</button>
    </form>
  );
};

export default ClockInClockOut;
