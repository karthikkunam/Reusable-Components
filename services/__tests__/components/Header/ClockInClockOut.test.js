import React from 'react';
import ClockInClockOut from '../../../src/components/Header/headerComponents/ClockInClockOut';
import { shallow } from 'enzyme';

describe('ClockInClockOut Component', () => {
  it('renders correctly', () => {
    const component = shallow(<ClockInClockOut/>);
    expect(component).toMatchSnapshot();
  });
});

