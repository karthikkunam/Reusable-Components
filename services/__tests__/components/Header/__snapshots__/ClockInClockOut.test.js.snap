// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClockInClockOut Component renders correctly 1`] = `
<form
  className="form-inline logged-out"
>
  <button
    className="btn btn-sm btn-outline-primary clockin"
    disabled={true}
    type="submit"
  >
    CLOCK IN / OUT
  </button>
</form>
`;
